addons/mail/static/src/owl/components/composer_text_input/composer_text_input.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.component.ComposerTextInput" owl="1">
         <div class="o_ComposerTextInput">
-            <textarea t-ref="textarea"/>
+            <textarea class="o_ComposerTextInputTextArea" t-att-value="storeProps.composer.textInputContent" placeholder="Write something..." t-on-keydown="_onKeydownEditable" t-on-input="_onInputEditable" t-ref="textarea" />
```

Comment:
`o_ComposerTextInput_textarea`
convention is `o_<ComponentName>_<elementName>-<modifierName>`

Reply:
Done